.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -140,10 +140,6 @@ jobs:
             mode: 'patch'
             name: '7'
 
-          - config: 'bpf-next'
-            mode: 'patch'
-            name: '8'
-
```

Comment:
@lmb Did we lose this test config by accident? (same for https://github.com/cilium/cilium/pull/30653)